### PR TITLE
Making cards clickable in Ranking replies page

### PR DESCRIPTION
**Pull request by mehdi-zibout**
Users were having problem with the [...] button #1208 
My proposed solution is to remove it and make the card clickable
Recording 2023-02-09 at 12 24 33
I changed the sensor options so that the dragging will start after the pointer moves by 10 pixels. 
I have tested this in my phone, and a laptop screen touch and so far haven't found any conflict between dragging and clicking.

Only the text area is clickable, in case of a problem one can use the handle. 
image

### CODE REVIEW

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Can you investigate the cypress test failure? It seems to be for the random task type and not a random test failure.  Before we merge this I'd like to see that test pass.

**Comment by mehdi-zibout** [comment]
@fozziethebeat it was because cypress test clicks on the middle of the card (to get it focused so it can be dragged by keyboard), but that causes full text window to open. I changed the test to click on the left side and it passes now.

**Review by mehdi-zibout — COMMENTED** [review]
@fozziethebeat  it was because cypress test clicks on the middle of the card (to get it focused so it can be dragged by keyboard), but that causes full text window to open. I changed the test to click on the left side and it passes now.

File: website/src/components/CollapsableText.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  maxLength?: number;
-  isDisabled?: boolean;
-}) => {
+export const CollapsableText = ({ text, maxLength = 220 }: { text: ReactNode; maxLength?: number }) => {
   if (typeof text !== "string" || text.length <= maxLength) {
     return <>{text}</>;
   } else {
-    return <span>{text.substring(0, maxLength - 3)}...</span>;
+    const visibleText = text.substring(0, maxLength - 3);
+    const dots = visibleText[visibleText.length - 1] === "." ? ".." : "...";
```
Comment (AbdBarho) [inline website/src/components/CollapsableText.tsx:8]: There is no need for this check, it is not problem.

also, it does not handle all cases, what if your text ends with two dots? with three?

as said, not needed.

**Review by AbdBarho — COMMENTED** [review]
One final comment

File: website/src/components/CollapsableText.tsx (hunk 1 above)
Comment (mehdi-zibout) [reply website/src/components/CollapsableText.tsx]: Okay removed it.
I have a question in `SortableItem.tsx` we're using `useMemo` to change the background when the card is active. Isn't this overkill? we could use Tailwindcss `hover:` for this and for the cursor as well.

**Review by AbdBarho — APPROVED** [review]
Thanks

File: website/src/components/CollapsableText.tsx (hunk 1 above)
Comment (AbdBarho) [reply website/src/components/CollapsableText.tsx]: Yes, our code needs simplification, feel free to clean up if you find anything worthy, preferably in another PR.

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**